### ISSUE TITLE
HIV-Positive Patients on Antiretroviral Therapy Have an Altered Mucosal Intestinal but Not Oral Microbiome

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#251]
HIV-Positive Patients on Antiretroviral Therapy Have an Altered Mucosal Intestinal but Not Oral Microbiome – Jingjing Meng – Microbiology Spectrum
https://journals.asm.org/doi/10.1128/spectrum.02472-22

**Comment by eve10111** [comment]
Hello @SvetlanaUP , can you assign this paper to me please?. Thank you

**Comment by eve10111** [comment]
Hello @SvetlanaUP , yes it is ready. Thank you

**Comment by eve10111** [comment]
Hello @SvetlanaUP,  accepted. Thank you very much

**Comment by PreciousMike** [comment]
Hello @SvetlanaUP please kindly assign this paper to me.

Thanks.

**Comment by eve10111** [comment]
Hello @SvetlanaUP , can you kindly review my curation? Here's the link to access it= https://bugsigdb.org/Study_976 
thank you very much.

**Comment by Scholarpat** [comment]
Well done @eve10111, on your curation 😊.

Hi @SvetlanaUP,

Below are the changes I've made to this curation upon review:

- **Study design:** Changed from cross-sectional to case-control because HIV-positive and control subjects were the study subjects.

- **Experiment 1:** Altered body site, condition, group definition, statistical test, and LDA. Added antibiotic exclusion and excluded matched-on.

- **Experiment 1:** Fig. S1, which is not curatable, was initially curated. Changed this to Fig. 1C and curated its signatures accordingly.

- **Additional experiments:** Curated Fig. 3B and 3C, Fig. 4B and 4C, and Fig. 5B and 5C as additional 3 experiments.

Kindly awaiting your feedback. Thank you very much.
